test/test_bind_integer.rb

```diff
@@ -0,0 +1,41 @@
+require 'oci8'
+require File.dirname(__FILE__) + '/config'
+
+class TestBindTime < Minitest::Test
```

Comment:
Could you rename `TestBindTime` to `TestBindInteger`?
It conflicts with `TestBindTime` in `test/test_bind_time.rb`.